src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs

```diff
@@ -127,11 +127,7 @@ private async Task DoPostAsync(HttpClient client, HttpRequestMessage request)
         {
             using (var response = await client.SendAsync(request).ConfigureAwait(false))
```

Comment:
We can do away with assigning the response here too.

Reply:
Hi @MikeGoldsmith , your suggestion would be this:
```csharp
using (await client.SendAsync(request).ConfigureAwait(false))
{
     // disposing response}
```

Reply:
I don't think we need the using statement either.

For example:
```suggestion
await client.SendAsync(request).ConfigureAwait(false);
```

Reply:
just updated!